app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="blue">
+  <div class="container-sm">
+    <div class="clase-header2 blue d-flex justify-between align-end">
+      <a href="<%= categories_path %>" class="back-button"><i class="fa-solid fa-arrow-left" style="color: #ffffff;"></i></a>
+      <h1 class="category-title"><%= @category.name %></h1>
+      <a href="<%= categories_path %>" class="btn-login blue"><i class="fa-solid fa-magnifying-glass" style="color: #ffffff;"></i></a>
```

Comment:
- [ **OPTIONAL** ] For CSS best practices, I think it would be better if you don't use inline styling like you did in some views files